src/components/shop/transact.js

```diff
@@ -34,6 +34,13 @@ export const unequip = (scene, item) => {
     invCol.set({ ...item, state: "purchased" });
 };
 
+export const use = (scene, item) => {
+    const { manage } = scene.config.paneCollections;
+    const invCol = collections.get(manage);
+    const invItem = invCol.get(item.id);
+    invCol.set({ ...invItem, qty: invItem.qty - 1 });
```

Comment:
just wondering how we keep track of what has been used and how we inform the game of this. I'm still a fan of handing this off to the game to deal with :joy:

Reply:
I guess there could be like a "used consumables" collection that the game gets, but then it would literally have to read it and go clear it itself.

Reply:
Definitely a question that needs resolving. Felt like it was out of scope for this ticket & needed its own conversation, though.

Reply:
Outcome of postflight chat - Graham will spin up a ticket re this.